index.bs

```diff
@@ -4646,6 +4646,13 @@ Registry Definitions</h4>
 			The [=custodian=] may be the [=Working Group=], the [=Team=], or a delegated entity.
 			The [=custodian=] for all [=registry tables=] in a single [=registry=]
 			<em class=rfc2119>should</em> generally be the same entity.
+
+			If a [=custodian=] of a [=registry=] becomes defunct
+			(e.g., the relevant group is disbanded),
+			and the [=Working Group=] that owns the [=registry definition=] is itself also defunct,
```

Comment:
Feels like we should have a defined term for the concept of a WG that's no longer operating - "defunct" feels wrong here. Maybe this is also a bit stronger than "does not have an active Charter" since as we have seen, WGs may effectively continue to operate for a while during Charter renegotiations that go on beyond the previous Charter's end point, for a while anyway - extensions are not always granted synchronously, for practical reasons.

Reply:
perhaps we should replace 'defunct' with what we have as examples, "the custodian ceases to exist, or is unresponsive to repeated attempts to make contact"?

Reply:
§4.6 describes ["Chartered Group Closure"](https://www.w3.org/2023/Process-20230612/#GeneralTermination) - we could either define a team for "Closed Group" within that section, or just use link to that section here, e.g.:

```suggestion
			and the [=Working Group=] that owns the [=registry definition=] is itself <a href="#GeneralTermination">closed</a>,
```